### ISSUE TITLE
Z-Wave JS Call Service in Automations shows blank Target options

### ISSUE THREAD

**Issue by nsheldon** [home-assistant/frontend#13418]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

None of the Z-Wave JS (zwave_js.*) call service actions in the UI provide buttons in the Target row to select target devices, entities or areas.  See screenshot below showing the "Z-Wave: Ping a node" service as an example.

![Z-Wave JS call service action no target buttons](https://user-images.githubusercontent.com/2738776/185689286-c5394a9b-6073-4c93-9b31-40a8adf1a6f0.png)

### Describe the behavior you expected

The "Target" row of all other Call Service actions in the UI provide at least 1 button to click in order to select a target device, entity or area.  Without such a button, there is no way to add target devices outside of using YAML.

### Steps to reproduce the issue

1. In the Home Assistant UI (core 2022.8.6, frontend 20220802.0) click "Settings" then "Automation & Scenes".  Be sure the "Automations" tab is selected.
2. Click the "CREATE AUTOMATION" button and then click "Start with an empty automation."
3. In the "Actions" section, set the "Action type" to "Call service".4. 
4. In the "Service" pop-up menu, select any "Z-Wave" service.  In my example, I selected the "Z-Wave: Ping a node" service, but this issue can be replicated using any of the "Z-Wave" services.
5. Note that the "Targets" row that appears below has no option to select a target.

### What version of Home Assistant Core has the issue?

core-2022.8.6

### In which browser are you experiencing the issue with?

Safari 15.6.1, Firefox 103.0.2, Home Assistant iOS app

### Which operating system are you using to run this browser?

macOS 12.5.1, iOS 15.6.1

**Comment by kpine** [comment]
Even specifying a target in YAML mode does not work, as the CALL SERVICE button is disabled.

![image](https://user-images.githubusercontent.com/791794/187059289-bda95fcd-fc89-49e1-907e-aed44f67227c.png)

**Comment by nsheldon** [comment]
This appears to be fixed in the latest Home Assistant core and Z-Wave JS release.  It may have been fixed in a previous patch, but I haven't re-tested this issue until now.  However, I've been able to confirm this is no longer an issue with the following versions:

- Home Assistant OS 8.5
- Home Assistant Core 2022.9.1
- Supervisor 2022.08.6
- Frontend 20220907.0
- Z-Wave JS Add-on 0.1.70

I'm closing this issue as resolved.
